react/components/ProductDescription/index.js

```diff
@@ -1,35 +1,88 @@
-import React, { PureComponent } from 'react'
+import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl'
 
 import VTEXClasses from './CustomClasses'
 
+import './global.css'
+
 /**
  * Product Description Component.
- * Render the description of a product.
+ * Render the description of a product and it's technical specifications.
```

Comment:
Render the description and technical specifications of a product

Reply:
👍